### PR TITLE
chore(Portal): fix listing of all components

### LINKED ISSUE #2163: ListSummaryFromPages behaves strange
## 🐛 Bug Report

Bug in `ListSummaryFromPages` in branch `v10`.

The `ListSummaryFromPages` behaves strange and generates way too many empty(hidden) items when used.
It also seems to include "non related files"(examples like `DrawerList` and `ScrollView` in `/components`):

image

image

image

From time to time, it also seems to automagically style the first item in the list as a h1 or so(see Accordion in the example below):

image

## To Reproduce

- Go to /uilib/components/
- Go to /uilib/components/fragments
- Go to /uilib/extensions/

## Expected behavior

https://eufemia.dnb.no/uilib/components/

image

### PR DESCRIPTION
**Pull request by tujoworker**
Fixes #2163

### CODE REVIEW

File: packages/dnb-design-system-portal/src/shared/parts/ListComponents.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { useStaticQuery, graphql } from 'gatsby'
+import ListSummaryFromPages from './ListSummaryFromEdges'
```
Comment (langz) [inline packages/dnb-design-system-portal/src/shared/parts/ListComponents.tsx:3]:
```suggestion
import ListSummaryFromEdges from './ListSummaryFromEdges'
```

File: packages/dnb-design-system-portal/src/shared/parts/ListElements.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { useStaticQuery, graphql } from 'gatsby'
+import ListSummaryFromPages from './ListSummaryFromEdges'
```
Comment (langz) [inline packages/dnb-design-system-portal/src/shared/parts/ListElements.tsx:3]:
```suggestion
import ListSummaryFromEdges from './ListSummaryFromEdges'
```

File: packages/dnb-design-system-portal/src/shared/parts/ListExtensions.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { useStaticQuery, graphql } from 'gatsby'
+import ListSummaryFromPages from './ListSummaryFromEdges'
```
Comment (langz) [inline packages/dnb-design-system-portal/src/shared/parts/ListExtensions.tsx:3]:
```suggestion
import ListSummaryFromEdges from './ListSummaryFromEdges'
```

File: packages/dnb-design-system-portal/src/shared/parts/ListFragments.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { useStaticQuery, graphql } from 'gatsby'
+import ListSummaryFromPages, { ListEdges } from './ListSummaryFromEdges'
```
Comment (langz) [inline packages/dnb-design-system-portal/src/shared/parts/ListFragments.tsx:3]:
```suggestion
import ListSummaryFromEdges, { ListEdges } from './ListSummaryFromEdges'
```

**Review by langz — APPROVED** [review]
Thanks for fixing this, looks great 🥇 

This is nothing new, but locally(Might just be here on my machine?), the first element in the list seems to be bigger(h1) than the rest of the elements. It usually doesn't happen on the "first render", but if you scroll a bit and hover the 1st element a few times, it usually becomes bigger 🪄 🧙‍♂️ 

Screenshot 2023-03-29 at 11 33 19
Screenshot 2023-03-29 at 11 31 53
Screenshot 2023-03-29 at 11 31 25

**Comment by tujoworker** [comment]
Yep, have seen that too. Will look into this as well.
Have added e2e tests now as well.

File: packages/dnb-design-system-portal/src/shared/parts/ListElements.tsx [hunk 5]
```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { useStaticQuery, graphql } from 'gatsby'
+import ListSummaryFromEdges from './ListSummaryFromEdges'
+
+export default function ListComponents() {
```
Comment (langz) [inline packages/dnb-design-system-portal/src/shared/parts/ListElements.tsx:5]:
```suggestion
export default function ListElements() {
```

File: packages/dnb-design-system-portal/src/shared/parts/ListFragments.tsx [hunk 6]
```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { useStaticQuery, graphql } from 'gatsby'
+import ListSummaryFromEdges, { ListEdges } from './ListSummaryFromEdges'
+
+export default function ListComponents() {
```
Comment (langz) [inline packages/dnb-design-system-portal/src/shared/parts/ListFragments.tsx:5]:
```suggestion
export default function ListFragments() {
```

**Pull request merged**